### DISCUSSION TITLE
How to use extends?

### DISCUSSION THREAD

**Discussion by matiasseeber** [prisma/prisma#19934] [Q&A] [answered]
Hi!,

Do i have to do something specific to call the $extends method?, because its simply failing to recognize the method as valid, i can use $use, $on and so on, but not extends.

Thank you.

import { Prisma,PrismaClient } from '@prisma/client';

const prisma = new PrismaClient().$extends(
    
);

**Comment by nurul3101** [comment, marked as answer]
Hello @matiasseeber 👋 

Which version of Prisma are you using?

If you are using a version below `4.16.0` then you would need to enable clientExtensions as a preview feature: [Reference](https://www.prisma.io/docs/concepts/components/prisma-client/client-extensions).

Here's an example of how you can enable the clientExtensions preview feature by mentioning it in `schema.prisma` file
```
generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["clientExtensions"]
}
```

if you are using prisma version above `4.16.0` then you shouldn't need to enable this preview feature.

**Reply by matiasseeber** [reply]
i updated the dependency an it worked, Thanks!
